### ISSUE TITLE
Google docstrings, hover preview for args when multiple words in parenthesis

### ISSUE THREAD

**Issue by zduvall** [microsoft/pylance-release#4395]
<h3> Environment data </h3>

-   Language Server version: 2023.5.30
-   OS and version: darwin x64
-   python.analysis.indexing: true
-   python.analysis.typeCheckingMode: off

<h3> Code Snippet </h3>

```python
def func(a: str = None, b: bool = None, c: int = None) -> None:
    """This is a description of the function

    Args:
        a (str, optional): Description about a. Defaults to None.
        b (bool, optional): Description about b. Defaults to None.
        c (int, optional): Description about c. Defaults to None.
    """
    pass
```

### Expected:

Args should each be on own line, such as when there are not multiple words inside "`(...)`" like this screenshot:

![good-hover](https://github.com/microsoft/pylance-release/assets/71565231/e6416d47-6259-4bce-8e3f-ad6dc5d041d8)

### Actual:

Args all collapse into a single line:

![bad-hover](https://github.com/microsoft/pylance-release/assets/71565231/a9177bc6-bffb-4866-a437-eeedf54d24d7)

For further context, this follows the [google docstring format](https://github.com/NilsJPWerner/autoDocstring/blob/HEAD/docs/google.md) that [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) uses for optional arguments.

Thoughts? Thanks!

**Comment by shiftysheep** [comment]
Also if you have a complex type it will not display properly. `List[str]` is an example of one I ran into.

**Comment by rchiodo** [comment]
This issue has been fixed in prerelease version 2023.5.41, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023541-24-May-2023-PreRelease)
